Makefile

```diff
@@ -1,10 +1,15 @@
+# Angus: Love this. Great structure, well thought out. Impressive to see you thinking of this stuff.
+
 ifneq ("$(wildcard .env)","")
-  include .env
-  export
+	# Angus: fun fact: not every environment variable that can be expressed in a .env file can be included in a Makefile.
```

Comment:
I conducted a small test to verify your claim since I had never used multiline variables in my env file before, and you were right. Although you can use backslashes at the end of each line to create a multiline variable, it introduces unwanted spaces when there are newlines. Consequently, I discovered that the only effective approach is to run a bash script beforehand, although it does not directly use the env file. This discovery is quite intriguing! 🧐 Thank you for the tips! 😀 https://gist.github.com/naofel1/bb9b497b11707be79db381a00c81e17a

Reply:
Yeah, it's annoying. For deployment, you'd typically handle all of this with a proper secrets manager, Kubernetes ConfigMaps, etc., depending on your stack. But in dev (and for small projects), it would be nice to have the convenience of env files. Of course, if you're confident that you won't have any multiline variables (reading a PEM string from a file is definitely better than pasting it into .env), there's no problem!

Reply:
I've just discovered that the `docker` command now supports the `--env-file` flag! It used to be `docker-compose`-only. This makes passing many env vars to Docker much easier.